resources/views/layouts/master.blade.php

```diff
@@ -6,6 +6,11 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    @if (isset($global_vars->custom_font_embed) || isset($global_vars->custom_font_name))
+      {!! $global_vars->custom_font_embed !!}
```

Comment:
What's the font embed look like here? If it's always the same snippet with just a variable "account ID" or some other identifier, I'd vote to include the just the ID in the variable (included here via  `{{ ... }}` escaped tags) and have the Typeset snippet in the actual Blade template (so we're not allowing admins to include _any_ unescaped markup on the page).